### ISSUE TITLE
iOS Simulator Builds damaged error on MacOS Catalina

### ISSUE THREAD

**Issue by 509dave16** [expo/expo-cli#1197]
## Description

The most recent iOS simulator builds that I have created are all damaged. I am able to launch the app on the simulator, but it fails to proceed past the splash screen.

## Expected Behavior

After running `xcrun simctl install booted <app path>`, I should be able to open the app and proceed past the splash screen.

## Observed Behavior

After running `xcrun simctl install booted <app path>`, I am stuck at the splash screen after launching the app. A system warning prompt is displayed to me saying that the app is damaged.

![Screen Shot 2019-11-10 at 10 40 03 PM](https://user-images.githubusercontent.com/2925882/68566605-ae153c00-040b-11ea-8c1c-daf364a4dcf4.png)

![Screen Shot 2019-11-10 at 10 52 59 PM](https://user-images.githubusercontent.com/2925882/68567048-e406f000-040c-11ea-91ef-2592c276d91b.png)

## Environment
```
Expo CLI 3.4.1 environment info:
    System:
      OS: macOS 10.15.1
      Shell: 5.7.1 - /bin/zsh
    Binaries:
      Node: 12.9.1 - ~/.nvm/versions/node/v12.9.1/bin/node
      Yarn: 1.12.3 - /usr/local/bin/yarn
      npm: 6.12.1 - ~/.nvm/versions/node/v12.9.1/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Android Studio: 3.4 AI-183.6156.11.34.5692245
      Xcode: 11.2/11B52 - /usr/bin/xcodebuild
    npmPackages:
      @types/react: 16.8.3 => 16.8.3
      @types/react-native: 0.60.13 => 0.60.13
      @types/react-navigation: ^1.5.15 => 1.5.15
      expo: ^34.0.1 => 34.0.4
      react: 16.8.3 => 16.8.3
      react-native: https://github.com/expo/react-native/archive/sdk-34.0.0.tar.gz => 0.59.8
      react-navigation: ^1.0.0-beta.21 => 1.6.1
    npmGlobalPackages:
      expo-cli: 3.4.1
```
Also specify:

- Phone/emulator/simulator platform, model and version: `Simulator - iPhone 11 Pro Max - 13.2`

## Reproducible Demo

1. **Download** zipped app bundle from here: https://exp-shell-app-assets.s3.us-west-1.amazonaws.com/ios/%40foodbarrio/foodbarrio-qa-3175d259-0256-49c0-a707-775191c1b9da-simulator.tar.gz
2. **Unzip** foodbarrio-qa-3175d259-0256-49c0-a707-775191c1b9da-simulator.tar.gz
3. **Launch** one simulator
4. **Run** `xcrun simctl install booted foodbarrio-qa.app
5. **Open** the `foodbarrioQA` app on the simulator
6. **Encounter** the observed behavior as mentioned above in the **Observed Behavior** section

**Comment by brentvatne** [comment]
this seems to be caused by catalina, you can work around it for now by doing this: https://apple.stackexchange.com/questions/372084/macos-catalina-app-is-damaged-and-cant-be-opened-you-should-move-it-to-the-t/372208#372208

we will look into having a command to do this for you in expo-cli

**Comment by brentvatne** [comment]
some discussion internally about it:

<img width="441" alt="Screen Shot 2019-11-14 at 12 36 20" src="https://user-images.githubusercontent.com/90494/68894112-780ddb80-06db-11ea-9018-006f9adf028c.png">
<img width="431" alt="Screen Shot 2019-11-14 at 12 36 28" src="https://user-images.githubusercontent.com/90494/68894118-793f0880-06db-11ea-9eeb-fda5fb842b37.png">

**Comment by 509dave16** [comment]
@brentvatne Thank you for the reply!

I had previously used a `sudo xattr` command with no luck from another article.
However the variation in the stackoverflow post you referenced works like a charm.

Before your post I resorted to [disabling Gatekeeper](https://medium.com/@panayiotisgeorgiou/how-to-fix-damaged-app-message-on-macos-sierra-f11dbf683e7a) on my Mac which isn't the greatest idea :(

**Comment by byCedric** [comment]
I would agree with Ville about the readability/discoverability of `expo build:ios|android --install`. We could also add the `--install` flag, and "imply" that the build type is simulator? But overall, I like it. Your primary action is building, and secondary (_after the build is done_) install it. 

But as stated, you have to wait until it's finished. That might take a couple of minutes and do something else. I know myself a bit and I could see myself closing the simulator by accident during this waiting time. I guess it's also nice to add the extra command, decoupled from building, to install the latest build to your simulator. I like the `expo install:ios|android` structure, although it's not related to `expo install`. Don't think we should mix this with the `expo client:ios` or `expo client:install:ios` commands, to avoid having client/app ambiguity?

I can get started on a command or script that pulls the build from remote and install it in the simulator. But I think it's wise to decide on the flags or extra commands we want.

Some things that came to mind:
- `--no-wait` cannot be used with `expo build:ios|android --install`, it **must** wait until the build is completed to be able to install it. (should error out)
- If we imply the simulator type, `--type=archive` cannot be used with `expo build:ios --install`. (should error out)
- If we add `expo build:android --install`, we also imply that `--type=app-bundle` cannot be used. (should error out)
- It might be a good idea to check if the simulator / android device is ready before start building with `expo build:ios|android`. I would be annoyed if I waited for the build, only to find out I forgot to run the simulator.  This could also be less of a problem with the `expo install:ios|android` command.

**Comment by brentvatne** [comment]
@byCedric - maybe let's keep this simple as possible for now: when you run the `expo build:ios -t simulator` command it should tell you to run a specific command once it's completed to install it to the simulator. that command would download whatever the most recent simulator build is, install it and open it

**Comment by Hirbod** [comment]
TLDR:
Download simulator build, extract tar.gz, open terminal, and call

```xattr -cr path/to/your.app```

Drag & Drop that file into your simulator. It will start. Annoying, but ok for now. Should've stayed on Mojave^^

**Comment by brentvatne** [comment]
locking this issue for now because @Hirbod described a workaround in the above comment for now and it should be the easiest thing for people to find
